config/v1/feature_gates.go

```diff
@@ -559,4 +559,11 @@ var (
 					productScope(ocpSpecific).
 					enableIn(TechPreviewNoUpgrade).
 					mustRegister()
+
+	FeatureGateMetricsCollectionProfiles = newFeatureGate("MetricsCollectionProfiles").
+						reportProblemsToJiraComponent("Monitoring").
+						contactPerson("rexagod").
+						productScope(ocpSpecific).
+						enableIn(Default, TechPreviewNoUpgrade).
```

Comment:
To be accessible by default, we need to see automated testing results (99% pass over two weeks) or have a QE ack.

If you remove the Default, it's mergeable immediately.